main.go

```diff
@@ -91,39 +91,28 @@ func main() {
 				Destination: &commands.VisorLogFlags.LogLevelNamed,
 			},
 			&cli.BoolFlag{
-				Name:        "tracing",
```

Comment:
@placer14 heads up, this changes the flags for setting up tracing a little bit. Details of the change are in this file.